packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2101,4 +2101,40 @@ moduleFor('Components test: curly components', class extends RenderingTest {
 
     this.assertText('initial value');
   }
+
+  ['@htmlbars a two way binding flows upstream when set during init (#11192)']() {
```

Comment:
this description implies change propagation happens during init, i believe (if i understand correctly, but correct me if im wrong) it means that properties set during init, can later change and propogate change events.